### PR TITLE
feat: emit projection format files

### PR DESCRIPTION
**Pull request by yreynhout**
This PR introduces the `Wegsegment.prj` and the `Wegknoop.prj` files in both the download for editing purposes as well as the download to compose the product from.

### CODE REVIEW

File: src/RoadRegistry.BackOffice.Api/ZipArchiveWriters/ProjectionFormatFileZipArchiveWriter.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            var prjEntry = archive.CreateEntry(_filename);
+            using (var prjEntryStream = prjEntry.Open())
+            using (var prjEntryStreamWriter = new StreamWriter(prjEntryStream, _encoding))
+            {
+                await prjEntryStreamWriter.WriteAsync(StaticFileContents);
+                await prjEntryStreamWriter.FlushAsync();
+            }
+        }
+
+        public static readonly string StaticFileContents = @"PROJCS[""Belge_Lambert_1972"",GEOGCS[""GCS_Belge_1972"",DATUM[""D_Belge_1972"",SPHEROID[""International_1924"",6378388.0,297.0]],PRIMEM[""Greenwich"",0.0],UNIT[""Degree"",0.0174532925199433]],PROJECTION[""Lambert_Conformal_Conic""],PARAMETER[""False_Easting"",150000.01256],PARAMETER[""False_Northing"",5400088.4378],PARAMETER[""Central_Meridian"",4.367486666666666],PARAMETER[""Standard_Parallel_1"",49.8333339],PARAMETER[""Standard_Parallel_2"",51.16666723333333],PARAMETER[""Latitude_Of_Origin"",90.0],UNIT[""Meter"",1.0]]";
```
Comment (yreynhout) [inline src/RoadRegistry.BackOffice.Api/ZipArchiveWriters/ProjectionFormatFileZipArchiveWriter.cs:34]: The content of all these files is always the same.

File: src/RoadRegistry.Wms.Projections/IStreetNameCache.cs [hunk 2]
```diff
@@ -14,48 +11,4 @@ public interface IStreetNameCache
         Task<long> GetMaxPositionAsync(CancellationToken token);
         Task<Dictionary<int, string>> GetStreetNamesByIdAsync(IEnumerable<int> streetNameIds, CancellationToken token);
     }
-
-    public class StreetNameCache : IStreetNameCache
```
Comment (yreynhout) [inline src/RoadRegistry.Wms.Projections/IStreetNameCache.cs:18]: One file, mostly one type - helps with discovering.

File: test/RoadRegistry.Tests/BackOffice/Api/ZipArchiveWriters/ForEditor/GradeSeparatedJunctionArchiveWriterTests.cs [hunk 3]
```diff
@@ -45,7 +45,7 @@ public async Task WithEmptyDatabaseWritesArchiveWithExpectedEntries()
             var db = await _fixture.CreateDatabaseAsync();
             var context = await _fixture.CreateEditorContextAsync(db);
 
-            await new ZipArchiveScenario<EditorContext>(_fixture, sut)
+            await new ZipArchiveScenario<EditorContext>(_fixture.MemoryStreamManager, sut)
```
Comment (yreynhout) [inline test/RoadRegistry.Tests/BackOffice/Api/ZipArchiveWriters/ForEditor/GradeSeparatedJunctionArchiveWriterTests.cs:48]: Changed contract to only use that which we rely on, nothing more.

File: src/RoadRegistry.Wms.Projections/IStreetNameCache.cs (hunk 2 above)
Comment (koenmetsu) [reply src/RoadRegistry.Wms.Projections/IStreetNameCache.cs:18]: Fine with me, but what are the exceptions?

File: test/RoadRegistry.Tests/BackOffice/Api/ZipArchiveWriters/ForEditor/GradeSeparatedJunctionArchiveWriterTests.cs (hunk 3 above)
Comment (koenmetsu) [reply test/RoadRegistry.Tests/BackOffice/Api/ZipArchiveWriters/ForEditor/GradeSeparatedJunctionArchiveWriterTests.cs:48]: :+1:

**Review by koenmetsu — COMMENTED** [review]
LGTM :+1:

File: src/RoadRegistry.Wms.Projections/IStreetNameCache.cs (hunk 2 above)
Comment (yreynhout) [reply src/RoadRegistry.Wms.Projections/IStreetNameCache.cs:18]: Delegates, private types, mostly.

**Pull request merged**